rviz_marker_tools/package.xml

```diff
@@ -9,9 +9,26 @@
 
 	<buildtool_depend>catkin</buildtool_depend>
 
-	<depend>visualization_msgs</depend>
-	<depend>geometry_msgs</depend>
-	<depend>roscpp</depend>
-	<depend>tf2_eigen</depend>
-	<depend>liburdfdom-dev</depend>
+	<build_depend>eigen</build_depend>
+	<build_export_depend>eigen</build_export_depend>
+
+	<build_depend>geometry_msgs</build_depend>
+	<build_export_depend>geometry_msgs</build_export_depend>
+	<exec_depend>geometry_msgs</exec_depend>
+
+	<build_depend>liburdfdom-headers-dev</build_depend>
+
+	<build_depend>roscpp</build_depend>
+	<exec_depend>roscpp</exec_depend>
```

Comment:
Why is `build_export_depend` missing here?

Reply:
because the header exposes no roscpp include - only the source uses it [once](https://github.com/moveit/moveit_task_constructor/blob/184d39897ae28158e57800535f50ace93d0a1016/rviz_marker_tools/src/marker_creation.cpp#L324) - and arguably that line should actually throw instead of printing to the log.